pensieve/cli.py

```diff
@@ -64,7 +64,11 @@ def convert(self, value, param, ctx):
 experiment_slug_option = click.option(
     "--experiment_slug",
     "--experiment-slug",
-    help="Experimenter or Normandy slug of the experiment to rerun analysis for",
+    help="Experimenter or Normandy slug of the experiment to (re)run analysis for",
+)
+
+secret_config_file_option = click.option(
+    "--i-solemnly-swear-i-am-up-to-no-good", "config_file", type=click.File("rt"), hidden=True
```

Comment:
The name `config_file` is just used as the function argument; it's not exposed to the CLI since it doesn't have any leading dashes.

The Click docs are not very clear on this point.